vendor/assets/stylesheets/dvl/core/lists.scss

```diff
@@ -6,25 +42,20 @@ ul, ol {
   list-style: none;
 }
 
+ul.formatted_list {
+  @include formatted_list_ul;
+}
+
+ol.formatted_list {
+  @include formatted_list_ol;
+}
+
 .rendered_from_wysiwyg,
-.wysiwyg_wrapper {
-  ul, ol {
-    list-style-position: inside;
-    margin-left: 1rem;
-  }
+.redactor_wysiwyg {
   ul {
-    list-style-type: disc;
+    @include formatted_list_ul;
   }
   ol {
```

Comment:
I could add them back in, but do we ever use them?

On Thu, Jul 2, 2015 at 10:54 AM, Adam Becker <notifications@github.com>
wrote:

> In vendor/assets/stylesheets/dvl/core/lists.scss
> <https://github.com/dobtco/dvl-core/pull/105#discussion_r33804311>:
>
> >    ul {
> > -    list-style-type: disc;
> > +    @include formatted_list_ul;
> >    }
> >    ol {
>
> Are we losing support for nested ols?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dvl-core/pull/105/files#r33804311>.
>